server/lib/nl/detection.py

```diff
@@ -171,6 +177,16 @@ class CorrelationClassificationAttributes(ClassificationAttributes):
   correlation_trigger_words: str
 
 
+@dataclass
+class TimeDeltaClassificationAttributes(ClassificationAttributes):
+  """Time Delta classification attributes."""
+  time_delta_type: List[TimeDeltaType]
```

Comment:
nit: if its a list, use time_delta_types ?

Reply:
Renamed to time_delta_types.